src/components/RoomNameInput/index.native.tsx

```diff
@@ -51,18 +68,18 @@ function RoomNameInput({isFocused, autoFocus, disabled, errorText, forwardedRef,
     );
 }
 
-RoomNameInput.propTypes = roomNameInputPropTypes.propTypes;
-RoomNameInput.defaultProps = roomNameInputPropTypes.defaultProps;
 RoomNameInput.displayName = 'RoomNameInput';
 
-const RoomNameInputWithRef = React.forwardRef((props, ref) => (
-    <RoomNameInput
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...props}
-        forwardedRef={ref}
-    />
-));
+function RoomNameInputWithRef(props: Omit<RoomNameInputProps, 'forwardedRef'>, ref: ForwardedRef<BaseTextInputRef>) {
+    return (
+        <RoomNameInput
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+            forwardedRef={ref}
```

Comment:
Can you remove `RoomNameInputWithRef` completely, and remove forwardedRef property in RoomNameInputProps?

Reply:
What is the reason behind removing `forwardedRef` prop?